system/src/ble_control_request_channel.cpp

```diff
@@ -1045,31 +1036,18 @@ size_t BleControlRequestChannel::readSome(char* data, size_t size) {
     return size;
 }
 
-int BleControlRequestChannel::connected(const ble_connected_event_data& event) {
```

Comment:
Let's make sure that the BLE control interface still functions correctly for all the weird edge-cases we've had with certain phones not supporting certain MTUs or intervals or data length extensions.